dspace/config/local.cfg.EXAMPLE

```diff
@@ -134,6 +134,14 @@ db.password = dspace
 # Recipient for new user registration emails (defaults to unspecified)
 #registration.notify =
 
+# An option is added to disable the mailserver. By default, this property is set to false
+# By setting mail.server.disabled = true, DSpace will not send out emails.
+# It will instead log the subject of the email which should have been sent
+# This is especially useful for development and test environments where production data is used when testing functionality.
+#mail.server.disabled = false
+# If the mail.server.fixedRecipient is not empty then only in the case mail.server.disabled = true all email will be sent to this email address.
+#mail.server.fixedRecipient =
```

Comment:
if you consider to change the configuration naming, then, this documentation should be also updated.